src/components/pages/coach/GenerateReport/index.jsx

```diff
@@ -348,11 +372,11 @@ class GenerateReport extends React.Component{
 
     sendData = (result) => {
         let values = {
-            "uid": this.state.user.id,
+            "uid": this.state.uid,
             "data": JSON.stringify(result)
         }
         this.props.send({
-            variables: { 
+            variables: {
                 "token": localStorage.getItem("fprint"),
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋